src/main/java/com/code_intelligence/jazzer/driver/FuzzTargetRunner.java

```diff
@@ -55,7 +55,9 @@
  */
 public final class FuzzTargetRunner {
   static {
-    AgentInstaller.install(Opt.hooks);
+    if(!Opt.isAndroid) {
+      AgentInstaller.install(Opt.hooks);
```

Comment:
This isn't just starting the agent, but also adds certain classes to the bootstrap classpath that instrumented classes may access. Is that something that can be done on Android? 

It may only be necessary when system classes are instrumented though, so it might not even apply to Android.

Reply:
We are not able to use ByteBuddy on Android, so ByteBuddyAgent.install() will fail. Instead of loading in the class through the Bootstrap jar I have added the needed deps to android_binary that builds "jazzer_standalone_android".